lib/logjam_agent/rack/logger.rb

```diff
@@ -99,6 +99,7 @@ def before_dispatch(request, env, start_time)
         end
         logjam_fields.merge!(:ip => ip, :host => @hostname)
         logjam_fields.merge!(extract_request_info(request))
+        logjam_fields.merge!(:wait_time_ms => wait_time_ms)
```

Comment:
Needs to use symbol :wait_time, not :wait_time_ms. This is fixed in  https://github.com/skaes/logjam_agent/commit/3fba8263aa0ce3a6a83a17afee634eba58b6252a